package.json

```diff
@@ -84,7 +84,8 @@
         "minimist": "1.2.0",
         "mustache": "2.3.0",
         "request": "2.83.0",
-        "swagger-js-codegen": "1.12.0",
+        "swagger-js-codegen-formdata": "^0.15.4",
+        "upgrade": "^1.1.0",
```

Comment:
@kastentx I think that `upgrade` module is not used in the code. Could you remove it if it is mistakenly added? (Could you teach me about the reason if it is needed?)

Reply:
Thank you. This was added in error.